legal-api/tests/unit/services/test_authorization.py

```diff
@@ -1293,82 +1222,40 @@ def mock_auth(one, two):  # pylint: disable=unused-argument; mocks of library me
     [
         # active business - staff user
         ('staff_active_cp', Business.State.ACTIVE, ['CP'], 'staff', [STAFF_ROLE],
-         expected_lookup([FilingKey.ADMN_FRZE,
-                          FilingKey.AR_CP,
-                          FilingKey.COA_CP,
-                          FilingKey.COD_CP,
-                          FilingKey.CORRCTN,
-                          FilingKey.COURT_ORDER,
-                          FilingKey.VOL_DISS,
-                          FilingKey.REGISTRARS_NOTATION,
-                          FilingKey.REGISTRARS_ORDER,
-                          FilingKey.SPECIAL_RESOLUTION])),
```

Comment:
this takes up more space but it was done intentionally to make things more readable.  can we revert all the places where we do this?